app/resources/en/translation/stoppage.json

```diff
@@ -3,12 +3,12 @@
-  "notExecutor": "You can&rsquo;t use the online service if you&rsquo;re not named as an executor or if you&rsquo;re applying on behalf of someone else.",
+  "noWill": "You can&rsquo;t apply for probate online because the person who died didn&rsquo;t leave a will.</p><p>Download and complete the <a rel=\"external noopener\" href=\"{applicationFormPA1A}\">paper application form (PA1A)</a>.</p><p>You can also <a rel=\"external noopener\" href=\"{guidance}\">download guidance</a> and find out where to <a rel=\"external noopener\" href=\"{registryInformation}\">send your completed form</a>.",
+  "notOriginal": "You can&rsquo;t apply for probate online because you don&rsquo;t have the original will of the person who died.</p><p>Download and complete the <a rel=\"external noopener\" href=\"{applicationFormPA1P}\">paper application form (PA1P)</a>.</p><p>You can also <a rel=\"external noopener\" href=\"{guidance}\">download guidance</a> and find out where to <a rel=\"external noopener\" href=\"{registryInformation}\">send your completed form</a>.",
+  "notExecutor": "You can&rsquo;t use the online service if you&rsquo;re not named as an executor or if you&rsquo;re applying on behalf of someone else.</p><p>Download and complete the <a rel=\"external noopener\" href=\"{applicationFormPA1P}\">paper application form (PA1P)</a>.</p><p>You can also <a rel=\"external noopener\" href=\"{guidance}\">download guidance</a> and find out where to <a rel=\"external noopener\" href=\"{registryInformation}\">send your completed form</a>.",
   "codicils": "You can&rsquo;t use the online service because there have been updates (&lsquo;codicils&rsquo;) made to the will.",
   "ihtNotCompleted": "You need to complete an <a rel=\"external noopener\" href=\"{ihtNotCompleted}\">inheritance tax form</a> before applying for probate. <br/>Your progress has been saved and will be kept for 6 months.",
-  "mentalCapacity": "You can&rsquo;t use the online service because not all executors are able to make their own decisions.",
-  "deathCertificate": "You need to have a <a rel=\"external noopener\" href=\"{deathCertificate}\">copy of the death certificate</a> before applying for probate.",
+  "mentalCapacity": "You can&rsquo;t use the online service as not all executors are able to make their own decisions.</p><p>Download and complete the <a rel=\"external noopener\" href=\"{applicationFormPA1P}\">paper application form (PA1P)</a>.</p><p>You can also <a rel=\"external noopener\" href=\"{guidance}\">download guidance</a> and find out where to <a rel=\"external noopener\" href=\"{registryInformation}\">send your completed form</a>.</p><p>You will need to complete a 'mental capacity form' for the executor who is unable to make their own decisions. Download it at: <a rel=\"external noopener\" href=\"https://www.gov.uk/wills-probate-inheritance/if-youre-an-executor\">https://www.gov.uk/wills-probate-inheritance/if-youre-an-executor</a>.",
```

Comment:
Can we change the apostrophe's around mental capacity form to be `&lsquo;` and `&rsquo;` please.

@shepda, I'm also not too happy about the level of markup (e.g. `<p>` tags) that's going into this content.  What are your thoughts?   In most other content files we adopt a paragraph by paragraph approach, so this example would have noWillParagraph1, noWillParagraph2 and noWillParagraph3 etc (see privacy.json for an example).  Using the above format could get tricky to maintain.

Reply:
Quotes replaced.